### ISSUE TITLE
TextInput should have ariaLabel tag

### ISSUE THREAD

**Issue by alembe** [carbon-design-system/carbon#5439]
<img width="595" alt="Screen Shot 2020-02-25 at 2 41 29 PM" src="https://user-images.githubusercontent.com/25492106/75285946-2c3ff780-57dd-11ea-90c8-7436fd43a3dc.png">

There's a DAP violation for the input field that needs a ariaLabel

**Comment by abbeyhrt** [comment]
Hi! @alembe! Thanks for making this issue! The `labelText` prop, that is a required prop, is intended to label the input so there should be no violation if that is defined. If you don't want to have a visible label, you can just supply an `aria-label` directly on `TextInput` and that also resolves the violation. Here's a simple [example](https://codesandbox.io/s/still-flower-49jk9?fontsize=14&hidenavigation=1&theme=dark).
